samples/Kconfig.mcumgr_bt_ota_dfu

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr OTA DFU over Bluetooth"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	imply MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	imply MCUMGR_CMD_IMG_MGMT
+	imply MCUMGR_CMD_OS_MGMT
+	imply MCUMGR_GRP_ZEPHYR_BASIC
+	imply MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	imply MCUMGR_SMP_REASSEMBLY_BT
+	imply NORDIC_QSPI_NOR
+	imply NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	depends on BT_PERIPHERAL
+	depends on SOC_SERIES_NRF53X
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Enable this option to include MCUmgr in the build which will enable
+	  the Bluetooth transport and image management group, which allows
+	  for firmware over the air update support of the device.
+
+if NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+
+config BT_PERIPHERAL_PREF_MIN_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_MAX_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_LATENCY
+	default 99
+
+config BT_PERIPHERAL_PREF_TIMEOUT
+	default 400
+
+config MCUMGR_SMP_BT_AUTHEN
+	default n
+
+config IMG_ERASE_PROGRESSIVELY
+	default y
+
+config MCUMGR_SMP_WORKQUEUE_STACK_SIZE
```

Comment:
Seems like it used to work fine with `CONFIG_MCUMGR_SMP_WORKQUEUE_STACK_SIZE=2048` before. May be worth to double check it (2048 is a default value, maybe there is no need to change it).

Reply:
you are right, it can be removed. I thought for some reason that DFU samples used the higher value for this configuration

Reply:
If it used to work with lower value then we should be fine I think